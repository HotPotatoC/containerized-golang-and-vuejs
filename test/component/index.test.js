const test = require('ava')
const Vue = require('vue')

const Component = require('../../src/components/Index.vue')

test('Rendering the Index Component', t => {
  const vm = new Vue(Component).$mount()
  const tree = {
    $el: vm.$el.outerHTML
  }
  t.snapshot(tree)
})
