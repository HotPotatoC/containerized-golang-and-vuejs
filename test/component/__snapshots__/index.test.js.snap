// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering the Index Component 1`] = `
Object {
  "$el": undefined,
}
`;
